examples/java-matter-controller/java/src/com/matter/controller/commands/common/Command.java

```diff
@@ -0,0 +1,243 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+package com.matter.controller.commands.common;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+public abstract class Command {
+  private String mName = null;
+  private String mHelpText = null;
```

Comment:
are these supposed to be nullable? Since we have constructors that do not initialize them, maybe make them Optional so that compiler helps developing safe code.

Reply:
make mHelpText Optional since it might be empty, mName is not nullable and always get set in constructor